src/stylesheets/_variables.scss

```diff
@@ -16,7 +16,9 @@ $brand-gray: (
 $font-sizes: (
   sm: rem(0.8125),
   md: rem(0.9375),
-  lg: rem(1.0625)
+  lg: rem(1.0625),
```

Comment:
@noor-ahmed Should we use `xxxrem`?
Because I'm unsure about this format you're using 😅